util.c

```diff
@@ -328,6 +328,8 @@ json_t *json_rpc_call(CURL *curl, const char *url,
 
 	curl_easy_setopt(curl, CURLOPT_NOSIGNAL, 1);
 	curl_easy_setopt(curl, CURLOPT_URL, url);
+	if (cert != NULL)
+		curl_easy_setopt(curl, CURLOPT_CAINFO, cert);
```

Comment:
This solution corresponds to solution number 2. of those [suggested by CURL developers here](http://curl.haxx.se/docs/sslcerts.html), but perhaps a one-time solution with no code changes could be along the lines of solutions 3. (append to the existing ca bundle).  This seems to amount to a single command to convert the dcrd.crt into the right format for appending:  `openssl x509 -inform DES -in dcrd.crt -out dcrd.pem -text`.  Then append to the bundle curl-ca-bundle.crt  Is this a workable solution?